todo_app/view_model.py

```diff
@@ -0,0 +1,24 @@
+from todo_app.item import Item
+
+
+class ViewModel:
+    def __init__(self, lists: list[tuple[str, str, list[Item]]]):
+        self._lists = lists
+
+    @property
+    def lists(self):
+        pass
+        return self._lists
+
+    @property
+    def doing_items(self):
+        return [ item for l in self.lists for item in l[2] if l[1] == "Doing" ]
```

Comment:
These properties are obviously a bit of a waste of time given that you don't use them (since you loop over all the lists) but I'm happy that this shows how we _could_ use them, and gives you clean unit tests for this module, so we'll not worry about that here!